src/pages/ImportWorkflow/importDockstoreWorkflow.ts

```diff
@@ -30,6 +37,16 @@ export const importDockstoreWorkflow = async ({ workspace, workflow, workflowNam
         sourceRepo: source,
       }
     }),
+    // If overwrite is true, attempt to delete any existing workflow with the given name.
+    // Do not error if no such workflow exists.
+    overwrite ?
+      workspaceApi.methodConfig(namespace, workflowName).delete().catch(err => {
```

Comment:
Out of curiosity do you know what might happen to the submissions once the method is deleted?

Reply:
Not off the top of my head. I didn't think to try that. I'll check it out though.

Reply:
The submission is still available in the job history. But there's no longer a link to the workflow configuration since the "configuration was updated or deleted since this submission ran".

<img width="983" alt="Screenshot 2023-02-23 at 2 39 22 PM" src="https://user-images.githubusercontent.com/1156625/221012537-b2a09a55-50c7-4b85-8404-a4a115424b3a.png">

Reply:
That's good and makes sense. Thank you for checking!